superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -579,6 +580,10 @@ const ResultSet = ({
       const expandedColumns = results.expanded_columns
         ? results.expanded_columns.map(col => col.column_name)
         : [];
+      const allowHTML = getItem(
+        LocalStorageKeys.SqllabIsRenderHtmlEnabled,
+        true,
```

Comment:
@soniagtm The default fallback value for this option should be `false` since it's an optional allowance.

Reply:
I thought that we wouldn't want to change the current behavior of the result table (FilterableTable), as it currently always renders results in HTML if applicable. That's why I set the default value to true.

Reply:
That makes sense. LGTM